deeptime/src/include/deeptime/markov/msm/tram/tram.h

```diff
@@ -245,12 +360,13 @@ struct TRAM {
 
     constexpr static dtype inf = std::numeric_limits<dtype>::infinity();
 
-    constexpr static dtype prior() { return 0.0; }
+    constexpr static dtype prior() { return tram::detail::prior<dtype>(); }
+
+    constexpr static dtype logPrior() { return tram::detail::logPrior<dtype>(); }
```

Comment:
no need to redefine this here, i'd just go with the `tram::detail::*prior` constexpr functions